templates/registry/registry-dpl.yaml

```diff
@@ -100,12 +96,17 @@ spec:
         - name: INTERNAL_TLS_TRUST_CA_PATH
           value: /etc/harbor/ssl/registry/ca.crt
         {{- end }}
-        {{- if .Values.persistence.imageChartStorage.azure.existingSecret }}
-        - name: REGISTRY_STORAGE_AZURE_ACCOUNTKEY
+        {{- if .Values.persistence.imageChartStorage.s3.existingSecret }}
+        - name: REGISTRY_STORAGE_S3_ACCESSKEY
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.persistence.imageChartStorage.s3.existingSecret.name }}
+              key: {{ .Values.persistence.imageChartStorage.s3.existingSecret.accessKey }}
+        - name: REGISTRY_STORAGE_S3_SECRETKEY
           valueFrom:
             secretKeyRef:
-              name: {{ .Values.persistence.imageChartStorage.azure.existingSecret }}
-              key: AZURE_STORAGE_ACCESS_KEY
```

Comment:
This config of line 103 ~ line 109 in `registry` container should also be kept and updated, instead of removing it.
Essentially, we need to update `REGISTRY_STORAGE_AZURE_ACCOUNTKEY` in both `registry` container and `registryctl` container, while in this PR, it removed the `REGISTRY_STORAGE_AZURE_ACCOUNTKEY` in `registry` container, which is not right, IMO.

Reply:
Fixed in [`ceeb717` (#1434)](https://github.com/goharbor/harbor-helm/pull/1434/commits/ceeb717816666e0ad23ceb82639fce64c28d186d)